Master-Game/World/Map.cs

```diff
@@ -25,7 +39,7 @@ public BaseTile TileAt(Point position)
             {
                 return null;
             }
-            return CurrentMap[position.X, position.Y];
+            return CurrentMap[position.Y, position.X]; //X - Horizontal , Y- vertical
```

Comment:
When the symbols were displayed, they were not the right orientation. For example, when I made a row of grass tiles, they were moving vertically instead of horizontally.